### DISCUSSION TITLE
How to join hpool.com

### DISCUSSION THREAD

**Discussion by padik11** [Chia-Network/chia-blockchain#5876] [Q&A]
I think a lot has been said about the delays in implementing official pools and the need to plot new plots. It is killing little farmers.

I made the decision to join hpool.com but I am not sure how to do it. I think there are many like me so I would like to start a discussion and hope for the help of people who have already joined hpool.com.

If you are using hpool.com please write a step by step process for joining this pool.

My first question:

Can I plot with the official GUI while connected to hpool.com?

Thank you and I hope everyone who has invested will start earning money.

**Comment by lvcivs** [comment]
Don't join hpool, there are serious security concerns with doing so.

**Reply by padik11** [reply]
How am I supposed to win with such a rapidly growing network? I have 100TB and nothing.

**Reply by Dizz02** [reply]
with patience. same thing happened with eth and btc...be patient
